ParseCSV/ParseCSV/Helper.cs

```diff
@@ -219,24 +219,44 @@ public static double GetTimeMinuteInterval(DateTime startTime, DateTime endTime)
         }
 
         // получает максимальное значение в списке в указанном диапазоне
-        public static double GetMaxInRange(List<double> inputList, Range range)
+        public static double GetMaxInRange(MyTable inputList, Range range, string typeOFPower = "ActivePower")
```

Comment:
Please try to avoid hardcoded values. Here you can use new enum for typeOFPower  instead of the string

Reply:
I will add 
enum TypeOfPower
    {
        ActivePower,
        ReactivePower
    }

Reply:
Enum with these values will be used as `TypeOfPower.ActivePower` or `TypeOfPower.ReactivePower`
I'd suggest you to remove repetitive parts, to have it like `PowerType.Active` and `PowerType.Reactive`